app/models/pydantic/datamart.py

```diff
@@ -202,3 +204,33 @@ def to_csv(
 
         csv_file.seek(0)
         return csv_file
+
+
+def _parse_area_of_interest(request: Request) -> AreaOfInterest:
```

Comment:
Should this still be prefixed with an underscore if we are importing into other modules?